### PR TITLE
Penny Update

### PR DESCRIPTION
**Pull request by MahdiBM**
A blog post on Penny.

### CODE REVIEW

**Comment by MahdiBM** [comment]
ah i think the CI fails because i've pushed the changed to my main branch which made the CI run at all. And then i don't have the credentials which makes it fail.

**Comment by MahdiBM** [comment]
@gwynne @0xTim feel free to give any suggestions

File: Content/posts/penny-update [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+date: 2023-05-16
```
Comment (0xTim) [inline Content/posts/penny-update:2]: This needs to be an ISO date IIRC

File: Content/posts/penny-update
Comment (0xTim) [inline Content/posts/penny-update:1]: This should be a .md file

File: Content/posts/penny-update [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+date: 2023-05-16 
+description: Penny Update - Keep Up With Swift's Evolution
+tags: penny, bot, discord
+author: Mahdi
+authorImageURL: /author-images/mahdi.jpg
+---
+# Penny
```
Comment (0xTim) [inline Content/posts/penny-update:8]: Let's add a more descriptive title - this is used on the homepage, in the open graph data etc

**Review by 0xTim — CHANGES_REQUESTED** [review]
Overall good, needs a few changes before publishing though

**Comment by 0xTim** [comment]
Hmm yeah CI shouldn't build if the blog is from a fork, but that's for another time

**Pull request merged**